WordPress/src/main/java/org/wordpress/android/ui/sitecreation/domains/compose/DomainItem.kt

```diff
@@ -72,36 +70,38 @@ fun DomainItem(uiState: DomainUiState) = with(uiState) {
                     Icon(
                         imageVector = Icons.Default.Check,
                         contentDescription = stringResource(string.selected),
-                        tint = MaterialTheme.colors.primary,
+                        tint = colors.primary,
                         modifier = Modifier.size(16.dp),
                     )
                 } else {
-                    variant?.dotColor?.let {
+                    tags.firstOrNull()?.dotColor?.let {
                         SolidCircle(size = 8.dp, colorResource(it))
                     }
                 }
             }
             Column(verticalArrangement = Arrangement.spacedBy(2.dp), modifier = Modifier.weight(1f)) {
                 Text(
                     text = domainName,
-                    color = MaterialTheme.colors.onSurface.takeIf { variant !is Unavailable } ?: SecondaryTextColor,
+                    color = colors.onSurface.takeIf { tags.none { it is Unavailable } } ?: SecondaryTextColor,
                     fontSize = PrimaryFontSize,
                     overflow = TextOverflow.Ellipsis,
                     maxLines = 1,
-                    modifier = Modifier.padding(bottom = TitleBottomPadding)
+                    modifier = Modifier.padding(bottom = 4.dp)
```

Comment:
I wonder why not to update the `TitleBottomPadding` in this case?

Reply:
> I wonder why not to update the TitleBottomPadding value in this case?

I removed it because I searched the file and didn't find any other usage of it 😃 , plus in the long run my view is that we'll refactor the current `sealed class Margin(val value: Dp)` and `4.dp` will become the "base" unit for spacing, so it should be quite easy to "fix" this when we focus on the design system basics.